feat: add instance configuration support (#448)
canonical/anbox-cloud-docs: howto/instance/configure-instance.md

(howto-configure-instance)=
# Configure an instance

AMS allows some configuration options of an instance to be changed. A full list of configuration options is available at {ref}`ref-ams-instance-configuration`

## Set confguration option

> To set an configuration option with the `amc` CLI you use the following command

Reviewer: ```suggestion
Use the `amc set` command to configure instance level options:
```

    amc set <instance id or name> <name> <value>

> For example to set the `security.delete_protected` option for the instance `test0`

Reviewer: ```suggestion
For example, to set the `security.delete_protected` option for the instance `test0`:
```

    amc set test0 security.delete_protected true

## Show current configuration options

To view the current set configuration options for an instance, you can run

>     amc show <instance id or name>

Reviewer: I think it is good to tell them how to view current configuration first and then tell them how to change it. Basically move this before the `amc set` part.

This will cater to both first-timers who will hopefully see a message that no configuration is set yet or the defaults (can you confirm this behavior?) and to repeat users who first check what configuration is set and then go about changing it.
